juriscraper/pacer/reports.py

```diff
@@ -238,7 +252,7 @@ def download_pdf(
                     f"Unable to get transcript at {url} in case "
                     f"{pacer_doc_id=}."
                 )
-            if "Sealed Document" in r.text:
+            if "Sealed Document" in r.text or "Under Seal" in r.text:
```

Comment:
We should add the detailed comment here too.

Reply:
Added comment with examples